weather_monitoring_operations/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views as weather_monitoring_views
+
+urlpatterns = [
+    path('weather/', weather_monitoring_views.get_weather_data, name="get_weather"),
```

Comment:
Can we run `black` formatting linter here. It will fix the EOF new line and convert  '' with ""s